db/models/shoppingItem.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const shoppingItemSchema = new Schema(
+  {
+    name: { type: String, required: true },
+    imageUrl: { type: String, required: true },
+    quantity: { type: Number, required: true },
+    category: { type: String, required: true },
+    comment: { type: String, required: true },
+  },
+  { collection: "shoppingItem" }
```

Comment:
```suggestion
  }
```

You don’t need this line unless you want explicit control over the mongodb collection name.

By default, mongoose automatically pluralizes the model name (shoppingItem) and uses that as the collection name → mongodb collection name: shoppingitems.

If you want to override this pluralization (force it to stay as "shoppingItem" instead of "shoppingitems"), then you could use `{ collection: "shoppingItem" }`..

Reply:
@Klausstille ,
I had an issue with the naming yesterday, because mongoose was creating another collection in MongoDB, that is why I "forced" to look to an specific collection. 

Can you explain a bit more about the naming that I have to use in MongoDB? so ideally always plural and lowercase?